elasticsearch/Dockerfile.centos7

```diff
@@ -0,0 +1,52 @@
+FROM centos:centos7
```

Comment:
We can not per @tdawson  who informed me OSBS only likes 'Dockerfile'.  Unless someone else can suggest otherwise.  I would prefer your desired convention too.